src/components/editor.tsx

```diff
@@ -16,6 +24,13 @@ export default function Editor({ savedata }: Props): React.JSX.Element {
     savedata.querySelector("achievementsList")!.innerHTML.split(" "),
   );
 
+  const [activeT1Filters, setActiveT1Filters] = useState(
+    [] as ChallengeFilter[],
+  );
+  const [activeT2Filters, setActiveT2Filters] = useState(
+    [] as ChallengeFilter[],
+  );
```

Comment:
We can use generics:

```suggestion
  const [activeT1Filters, setActiveT1Filters] = useState<ChallengeFilter[]>([]);
  const [activeT2Filters, setActiveT2Filters] = useState<ChallengeFilter[]>([]);
```